cmd/readiness/main.go

```diff
@@ -240,61 +240,11 @@ func isInReadyState(health health.Status) bool {
 			return true
 		}
 
-		if mongoDbServerHasStarted(health) {
-			// There should be only one entry reported for this Pod.
-			return processHealth.IsReadyState()
-		}
+		// Every time the process health is created by the agent,
```

Comment:
This is true only for versions of the agent that support this field; old agents won't write it.

Reply:
The field `lastMongoUp` is not related to the replication status, and it has been there since two years ago:

https://github.com/10gen/mms-automation/blame/bb72f74a22d98cfa635c1317e623386b089dc69f/go_planner/src/com.tengen/cm/healthcheck/status.go#L45

Reply:
Sorry, I meant `processHealth.IsReadyState()` which checks for the ReplicationStatus

Reply:
That's true, but we don't get to that call if replication status is `nil`, because that is checked right above